### PR TITLE
Ap/docu levelset

### PR DESCRIPTION
**Pull request by APaganini**
Added the level set example

### CODE REVIEW

**Comment by florianwechsung** [comment]
We now have code duplication. The levelset example can be found once in `examples/` and once in ` docs/source/` can this be avoided?

**Comment by APaganini** [comment]
Yes it can be avoided. I suggest to drop the one from`examples/`. Fine by you?

**Comment by florianwechsung** [comment]
I think someone who is new to the project is more likely to check the `examples` directory then the `docs` directory for an example. So I'd prefer to keep that one.

File: docs/source/ExampleLevelset/levelset.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+where :math:`f:\mathbb{R}^2\to\mathbb{R}` is a scalar function.
+In particular, we consider
+
+.. math::
+
+    f(x,y) = (x - 0.5)^2 + (y - 0.5)^2 - 0.5\,.
+
+The domain that minimizes this shape functional is a
+disc of radius :math:`1/\sqrt{2}` centered at the origin.
```
Comment (florianwechsung) [inline docs/source/ExampleLevelset/levelset.rst]: centered at (0.5, 0.5)

**Comment by APaganini** [comment]
I removed the duplication-of-code issue. Now `docs/source` contains only `*.rst` files (which is documentation). All `*.py`files are in `fireshape/examples`. There is still some redundancy with `fireshape/zoo`, but we should discuss the role of `zoo` in a different thread.

**Comment by APaganini** [comment]
@florianwechsung if you are happy with this pull request, please merge and close it.

**Comment by florianwechsung** [comment]
There is a `levelset_old.py`, can we get rid of it?

**Comment by APaganini** [comment]
I removed `levelset_old.py`

**Pull request merged**